src/localstorage.ts

```diff
@@ -1,7 +1,8 @@
-import * as cachingLayer from './cachinglayer';
 import * as log from './log';
 import * as eventHandling from './eventhandling';
 import {localStorageAvailable} from './util';
+import {RSNodes} from './common-interfaces';
+import {CachingLayerBase} from './cachinglayer-new';
```

Comment:
I don't think we have to add the "Base" suffix there. Extending from something automatically makes the other thing a "base" thing.